### PR TITLE
POC - footer clickable 

### PR DESCRIPTION
**Pull request by FrancoisGe**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/core/commands/command_palette.js [hunk 1]
```diff
@@ -267,6 +267,9 @@ export class CommandPalette extends Component {
             searchValue,
             activeElement: this.activeElement,
         });
+        if (this.inputRef.el) {
```
Comment (aab-odoo) [inline addons/web/static/src/core/commands/command_palette.js:270]: check with @Polymorphe57

File: addons/web/static/src/core/commands/command_service.js [hunk 2]
```diff
@@ -43,14 +43,20 @@ class DefaultFooter extends Component {
             .map((el) => ({ namespace: el[0], name: el[1].name }))
             .filter((el) => el.name);
     }
+
+    onClick(namespaces) {
+        this.props.search(namespaces);
```
Comment (aab-odoo) [inline addons/web/static/src/core/commands/command_service.js:48]: keep current search ?

File: addons/web/static/tests/core/commands/command_service_tests.js [hunk 3]
```diff
@@ -489,6 +489,74 @@ QUnit.test("footer displays the right tips", async (assert) => {
     );
 });
 
+QUnit.test("namespaces display in the footer are clickable", async (assert) => {
```
Comment (aab-odoo) [inline addons/web/static/tests/core/commands/command_service_tests.js:492]: displayed

File: addons/web/static/src/core/commands/command_palette.js [hunk 4]
```diff
@@ -271,20 +271,18 @@ export class CommandPalette extends Component {
...
-                this.debounceSearchByNamespace[namespace] = this.defaultDebounceSearch;
+        const namespaceConfig = this.configByNamespace[namespace] || {};
+        if (this.namespace !== namespace) {
+            if (this.lastDebounceSearch) {
+                this.lastDebounceSearch.cancel();
             }
+            this.lastDebounceSearch = debounce.apply(this, [
+                this.search,
+                namespaceConfig.debounceDelay || 0,
+            ]);
```
Comment (aab-odoo) [inline addons/web/static/src/core/commands/command_palette.js:285]: Why not
```js
this.lastDebounceSearch = debounce(() => this.search, namespaceConfig.debounceDelay || 0);
```
?

File: addons/web/static/src/core/commands/command_palette.js [hunk 5]
```diff
@@ -271,20 +271,18 @@ export class CommandPalette extends Component {
...
+            this.namespace = namespace;
         }
-        const debounceSearch =
-            this.debounceSearchByNamespace[namespace] || this.defaultDebounceSearch;
-        this.searchValuePromise = debounceSearch.apply(this, [value]).catch(() => {
+        this.searchValuePromise = this.lastDebounceSearch.apply(this, [value]).catch(() => {
```
Comment (aab-odoo) [inline addons/web/static/src/core/commands/command_palette.js:288]:
```js
this.searchValuePromise = this.lastDebounceSearch(value).catch(() => {
```

File: addons/web/static/tests/core/commands/command_service_tests.js (hunk 3 above)
Comment (aab-odoo) [reply addons/web/static/tests/core/commands/command_service_tests.js:493]: still :-)

**Review by aab-odoo — APPROVED** [review]
Good PR. The calls to debounce look unnecessary complicated, but I may be missing something

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

**Comment by FrancoisGe** [comment]
robodoo rebase-ff r+

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**